docs/dptp-triage-sop/logs/query.txt

```diff
@@ -0,0 +1,23 @@
+ fields @timestamp, msg, error, level, @message
+| sort @timestamp desc 
+| filter(level = "error" or level = "fatal")
+| filter(not(
+# exclusions we expect to keep forever
+	component="entrypoint" or component="sidecar" or component="clonerefs" or component="initupload" # these are logs from jobs, which we don't care about
+	or `kubernetes.pod_labels.prow.k8s.io/type`="presubmit" # these are logs from jobs, which we don't care about
```

Comment:
I think I ended up filtering out all job types: infra jobs on `app.ci` should have separate alerts. At minimum, when you don't care about `presubmit` you also don't care about `batch`.

Reply:
I didn't really know what would NOT be excluded by line 6 and would need this, as well